packages/wrangler/src/bundle-reporter.tsx

```diff
@@ -2,6 +2,9 @@ import { Blob } from "node:buffer";
 import { gzipSync } from "node:zlib";
 import { logger } from "./logger";
 import type { CfModule } from "./worker";
+import type { Metafile } from "esbuild";
+
+const ONE_MB = 1 << (10 * 2);
```

Comment:
Bit twiddling is not a common pattern in JS projects. Is there a way we can more explicitly represent 1MB here?